app/queries/notifications_resolver.rb

```diff
@@ -0,0 +1,46 @@
+class NotificationsResolver < ApplicationQuery
+  property :search
+  property :status, validates: { inclusion: { in: Notification::VALID_STATUS_TYPES } }, allow_blank: true
```

Comment:
Let's just read these from the related _type_?

```suggestion
  property :status, validates: { inclusion: { in: Types::NotificationStatusType.values.keys } }, allow_blank: true
```